ghost/i18n/locales/vi/portal.json

```diff
@@ -75,13 +75,13 @@
     "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Nếu bạn đã hoàn thành tất cả các bước kiểm tra này mà vẫn không nhận được email, hãy liên hệ với {{supportAddress}} để được hỗ trợ.",
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "Trong trường hợp nhận được lỗi vĩnh viễn khi cố gắng gửi bản tin, email sẽ bị vô hiệu hóa trên tài khoản.",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Trong ứng dụng email của bạn, hãy thêm {{senderEmail}} vào danh sách liên hệ của bạn. Điều này báo hiệu cho nhà cung cấp dịch vụ email của bạn rằng các email được gửi từ địa chỉ này là đáng tin cậy.",
-    "Invalid email address": "",
-    "Jamie Larson": "",
-    "jamie@example.com": "",
+    "Invalid email address": "Địa chỉ email không hợp lệ",
+    "Jamie Larson": "Hồ Nhất Duy",
+    "jamie@example.com": "duy@gmail.com",
     "Less like this": "Không thích lắm",
     "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Hãy chắc rằng email không đang trong hộp thư Spam hoặc Quảng cáo. Nếu đang vậy, chọn \"Đánh dấu không phải spam\" hoặc \"Chuyển tới Hộp thư đến\".",
     "Manage": "Quản lý",
-    "Maybe later": "",
+    "Maybe later": "Để sau",
```

Comment:
Would `Có lẽ sau này` be more appropriate here?

Reply:
In Vietnamese, `Có lẽ sau này` usually has a romantic meaning. Like a guy telling a girl he will marry her in the future :)) `Để sau` aka `Maybe later` is a more concise and accurate phrase.

Reply:
And this is why we don't rely on automated translations 😂

Thanks for the explanation 👍